rllib/utils/tf_utils.py

```diff
@@ -810,3 +812,230 @@ def print_warning():
             false_fn=lambda: tf.constant(0.0),
             true_fn=lambda: print_warning(),
         )
+
+
+def _unflatten(vector, shapes):
+    i = 0
+    arrays = []
+    for shape in shapes:
+        size = np.prod(shape, dtype=np.int_)
+        array = vector[i : (i + size)].reshape(shape)
+        arrays.append(array)
+        i += size
+    assert len(vector) == i, "Passed weight does not have the correct shape."
+    return arrays
+
+
+@DeveloperAPI
```

Comment:
Not sure if this is right, lmk @sven1977 @simonsays1980

As far as I can tell we don't even have public docs for the `tf_utils`, so not sure if they're used